flake/modules.nix

```diff
@@ -0,0 +1,80 @@
+{
+  inputs,
+  self,
+  lib,
+  ...
+}:
+{
+  flake = {
+    nixosModules.stylix =
+      { pkgs, ... }@args:
+      {
+        imports = [
+          (import "${self}/stylix/nixos" inputs)
+          {
+            stylix = {
+              inherit inputs;
+              paletteGenerator =
+                self.packages.${pkgs.stdenv.hostPlatform.system}.palette-generator;
+              base16 = inputs.base16.lib args;
+              homeManagerIntegration.module = self.homeModules.stylix;
+            };
+          }
+        ];
+      };
+
+    # Drop this alias after 26.05
+    homeManagerModules = lib.mkIf (lib.oldestSupportedReleaseIsAtLeast 2511) (
+      builtins.warn "stylix: flake output `homeManagerModules` has been renamed to `homeModules`" self.homeModules
+    );
```

Comment:
I think this may be inverted, I ran into this locally when reading the `homeManagerModules` output my mistake:

```
error: The option `flake.homeManagerModules' was accessed but has no value defined. Try setting the option.
```

Reply:
Or maybe something is off with how flake-parts handles freeform `flake.*` definitoins; e.g. maybe it's `lazyAttrsOf` instead of `attrsOf`?

Might need to move up the `mkIf`. Will investigate and PR.

Reply:
#1287